### ISSUE TITLE
Cut/copy and paste loses formatting in WYSIWYG

### ISSUE THREAD

**Issue by kantol** [taigaio/taiga-front#1280]
**Before adding an issue**
Please ensure that your issue is not already reported in our [issues list](https://tree.taiga.io/project/taiga/issues?order_by=-created_date).
If this issue was already reported, remember that you can upvote it to raise its importance.

**Do you want to request a *feature* or report a *bug*?**
Bug
**What is the current behavior?**
If you cut and paste from one description to another you lose your formatting in WYSIWYG

**If the current behavior is a bug, please provide the steps to reproduce.**
1. Select multi-line text in a description field in the WYSIWYG view
2. Use Ctrl+C to Copy multi-line text e.g.
This is an example of multiline text.
When you copy paste it runs together.
3. Within the same description field, use Ctrl-V to paste the text.
4. The pasted text loses the CR/LF.  For example:
This is an example of multiline text. When you copy paste it runs together.
5. This problem is even worse if the multi-line text has bullets. The bullets do not show up at all. 

**What is the expected behavior?**
Formatting should be maintained when copying & pasting.

**Is it happening in taiga.io or in your own instance?**
taiga,io

**What browser/version are you using?**
Chrome latest

**Are there any console errors *(Ctrl + F12)* in red?**
Yes. OTS Parsing Error: invalid version tag

Thanks for reporting!

**Comment by RobertBMerriman** [comment]
+1 This is quite annoying and really slows down adding stories

**Comment by zenichev** [comment]
+1 This is quite annoying and really slows down adding stories

**Comment by foliovision** [comment]
I thought this was really a problem as well until I realised that in Markdown mode, pasting works perfectly. The toggle between HTML and Markdown is cost free (i.e. all formatting is preserved).

Still it should be possible for Taiga to at least keep paragraphs when pasting into WYSIWYG even if stripping off the rest of the formatting. We did something similar with FCK Editor in Foliopress WYSIWYG. To prevent clients pasting in Microsoft Word HTML formatting (nightmare) by default, we made paste plain text, but did respect line breaks.
